src/sortStudents.ts

```diff
@@ -1,7 +1,52 @@
 // describe Student type
+interface Student {
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[]
+}
 // create and export SortType enum
+export enum SortType {
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades'
+}
 // create SortOrder type
+type SortOrder = 'asc' | 'desc';
+
+const averageGrade = (grades: number[]):number => {
+  return [...grades]
+    .reduce((sum: number, current: number) => sum + current) / grades.length;
+};
+
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  return [...students].sort((a, b) => {
+    switch (sortBy) {
+      case 'name':
+      case 'surname':
```

Comment:
use enum in cases everywhere

```suggestion
      case SortType.Name:
      case SortType.Surname:
```